<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVDBInternalLoadsLightsEditWidget</class>
 <widget class="QWidget" name="SVDBInternalLoadsLightsEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutMaster" columnstretch="0,1,0,0,0">
   <item row="2" column="0">
    <widget class="QLabel" name="labelLightsInput">
     <property name="text">
      <string>Power:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="comboBoxMethod"/>
   </item>
   <item row="3" column="4">
    <widget class="QToolButton" name="toolButtonRemovePowerManagementSchedule">
     <property name="icon">
      <iconset resource="../resources/SIM-VICUS.qrc">
       <normaloff>:/gfx/actions/24x24/minus.png</normaloff>:/gfx/actions/24x24/minus.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="toolButtonSelectSchedule">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QtExt::LanguageStringEditWidget1" name="lineEditName"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelDisplayName_2">
     <property name="text">
      <string>Power Management Schedule Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelCategory_2">
     <property name="text">
      <string>Method:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditManagementScheduleName">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelConvectiveFactor">
     <property name="text">
      <string>Convective Factor:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QtExt::ColorButton" name="pushButtonColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelDisplayName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QwtPlot" name="widgetPlot" native="true"/>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QLabel" name="labelLightsInputUnit">
     <property name="text">
      <string>Person</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QLabel" name="labelConvectiveFactorUnit">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditConvectiveFactor">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QtExt::ValidatingLineEdit" name="lineEditPower">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::LanguageStringEditWidget1</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_LanguageStringEditWidget1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
   <slots>
    <signal>editingFinishedSuccessfully()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QtExt::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">QtExt_ColorButton.h</header>
   <slots>
    <signal>colorChanged()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header location="global">qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/SIM-VICUS.qrc"/>
 </resources>
 <connections/>
</ui>
